tests/test_todo_parser.py

```diff
@@ -12,6 +12,22 @@ def count_issues_for_file_type(raw_issues, file_type):
             num_issues += 1
     return num_issues
 
+def get_issues_for_fields(raw_issues, fields):
+    matching_issues = []
+    for issue in raw_issues:
+        for key in fields.keys():
+            if getattr(issue, key) != fields.get(key):
+                break
+        else:
+            matching_issues.append(issue)
+    return matching_issues
```

Comment:
Essentially a generic version of `cont_issues_for_file_type()` that lets us find issues which match any arbitrary set of fields. These are AND-ed together. i.e. The issue's fields would need to match everything in `fields` in order to be part of the list that's returned.